external_services/lti.py

```diff
@@ -62,9 +62,9 @@ def __init__(self, service, user, instance, request, title, context_id=None, lin
             "launch_presentation_locale": get_language(),
             "launch_presentation_document_target":
                 "iframe" if exercise and exercise.open_in_iframe else "window",
-            "launch_presentation_return_url": settings.BASE_URL + instance.get_absolute_url(),
+            "launch_presentation_return_url": urljoin(settings.BASE_URL, instance.get_absolute_url()),
```

Comment:
This is basically the same as the `netloc + absolute_url`, but it's fine as is. This will be worked on in #431

Reply:
Yes, except the scheme. I wanted to keep the scheme there since it was previously there.

Reply:
> Yes, except the scheme. I wanted to keep the scheme there since it was previously there

Good point!